### ISSUE TITLE
Compilation fail on Debian Stretch (2)

### ISSUE THREAD

**Issue by TheWizIL** [ryancdotorg/brainflayer#46]
When running compiler, you get secp256k1 and compile it.

Problem it that compiling secp256k1 on Debian Stretch we need to set:

> ./configure --enable-openssl-tests=no

[Cannot build on Debian stretch #429](https://github.com/bitcoin-core/secp256k1/issues/429)

I try to build secp256k1 before running make on brainflayer, but brainflayer overwrite it.

**Comment by TheWizIL** [comment]
Log

> cd secp256k1; make
make[1]: Entering directory '/var/www/scripts/brainflayer/secp256k1'
gcc -I./ -g -O2 -Wall -Wextra -Wno-unused-function -c src/gen_context.c -o gen_context.o
gcc gen_context.o -o gen_context
./gen_context
  CC       src/libsecp256k1_la-secp256k1.lo
  CCLD     libsecp256k1.la
/usr/bin/ar: `u' modifier ignored since `D' is the default (see `U')
  CC       src/tests-tests.o
In file included from src/secp256k1.c:11:0,
                 from src/tests.c:17:
src/tests.c: In function ‘secp256k1_ecdsa_signature_parse_der’:
src/secp256k1.c:24:16: warning: nonnull argument ‘sig’ compared to NULL [-Wnonnull-compare]
     if (EXPECT(!(cond), 0)) { \

src/util.h:36:39: note: in definition of macro ‘EXPECT’
 #define EXPECT(x,c) __builtin_expect((x),(c))

**Comment by hevnsnt** [comment]
I am having the same issue on debian stretch

**Comment by brizzbane** [comment]
I'm running Debian Stretch, and everything works fine after following the suggestion in this comment:
https://github.com/ryancdotorg/brainflayer/issues/30#issuecomment-279176039

Remove all references to '-static', in the Makefile.  That is all I had to do.  (No changing SSL, or adding -lpthread, etc. --just remove references to -static).

At first I tried adding -lpthread as suggested elsewhere on the net to get it to compile.  But then I ran into issues where it would not run after it compiled.  So then I read the comment more closely.  I got rid of all changes to Makefile except for removing every -static.  It then compiles, and runs fine.
